go/logic/applier.go

```diff
@@ -49,6 +49,15 @@ func newDmlBuildResultError(err error) *dmlBuildResult {
 	}
 }
 
+func (this *Applier) setOptimizer(tx *gosql.Tx) error {
```

Comment:
@cyrinux the `setOptimizer` func-name feels a bit unclear about what it performs I think `setOptimizerSwitch` is a more descriptive func name. What do you think?